generators/sram-bist/src/test/scala/BistTopSpec.scala

```diff
@@ -474,11 +474,9 @@ class BistTopSpec extends AnyFlatSpec with ChiselScalatestTester {
           )
         )
       )(
-        new WithChiseltestSrams(ChiseltestSramFailureMode.none)
+        new freechips.rocketchip.subsystem.WithClockGateModel
       )
-    ).withAnnotations(Seq(WriteVcdAnnotation)) { c =>
-
-      // TODO: Test on SRAMs with smaller than 32 data width to make sure we aren't checking too many bits.
+    ).withAnnotations(Seq(VcsBackendAnnotation, WriteVcdAnnotation)) { c => 
```

Comment:
Did you change this? I would put it back to the original just so it can be tested locally as well.